typed/typed.go

```diff
@@ -70,7 +70,7 @@ func (dm Domain) FmtDefinitionEncoding(field string) (fmtEnc []*big.Int) {
 // Returns:
 // - *felt.Felt: a *felt.Felt with the value of str
 func strToFelt(str string) *felt.Felt {
-	var f = &felt.Zero
```

Comment:
This was leading to errors as it was overriding the felt. Zero value (e.g. line 77)